src/test/java/com/synopsys/integration/jenkins/JenkinsProxyHelperTest.java

```diff
@@ -0,0 +1,79 @@
+package com.synopsys.integration.jenkins;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import com.synopsys.integration.rest.proxy.ProxyInfo;
+
+public class JenkinsProxyHelperTest {
+
+    ProxyInfo proxyInfo;
+
+    static String includeHost = "www.include.com";
+    static String excludeHost = "www.exclude.com";
+
+    static Pattern includePattern = Pattern.compile(includeHost);
+    static Pattern excludePattern = Pattern.compile(excludeHost);
+
+    static final String url = "https://" + includeHost;
```

Comment:
url seems like something that should change test-to-test-- the fact that it's always the include URL tells me we never test if the exclude url gets filtered out. Seems like something we should test!